tesseract/tesseract_geometry/include/tesseract_geometry/mesh_parser.h

```diff
@@ -45,15 +48,22 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <assimp/Importer.hpp>
 #include <assimp/postprocess.h>
 
+#ifdef TESSERACT_ASSIMP_USE_PBRMATERIAL
+#include <assimp/pbrmaterial.h>
+#endif
+
 #include <console_bridge/console.h>
 
 #include <tesseract_common/types.h>
 #include <tesseract_common/resource.h>
 
 #include <regex>
+#include <boost/filesystem/path.hpp>
 
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
+#include <tesseract_geometry/impl/mesh_material.h>
+
 namespace tesseract_geometry
 {
 /**
```

Comment:
Recommend updating the documentation below.

Reply:
I have added argument documentation to the doxygen headers.